.travis.yml

```diff
@@ -31,22 +31,30 @@ matrix:
           os: linux
           dist: xenial
           env: TOXENV=py38
-        - python: 3.5
+        - python: "3.5"
           os: linux
           dist: xenial
           env: TOXENV=flake8
         - language: generic
           os: osx
-          osx_image: xcode8.3
+          osx_image: xcode8.3  # This is the latest working xcode image with osxfuse compatibility; later images come with an OS X version which doesn't allow kernel extensions
           env: TOXENV=py35
         - language: generic
           os: osx
           osx_image: xcode8.3
           env: TOXENV=py37
+        - language: generic
+          os: osx
+          osx_image: xcode11.3
+          env: TOXENV=py35 SKIPFUSE=true
```

Comment:
can we remove this, so we only have 2 macOS jobs in total?

Reply:
I'm sorry, I don't understand. Currently, there are 4 jobs with OS X, testing xcode8.3/xcode11.3 and py35/py37, respectively. Which ones do you want to remove to arrive at 2 OS X jobs in total?

Reply:
- old osx / xcode, old python
- new osx / xcode, new python

Reply:
:+1: